### ISSUE TITLE
[App Support]: Uma Musume Pretty Derby stuck on loading screen

### ISSUE THREAD

**Issue by Gabby790** [PlayCover/PlayCover#1276]
### Describe the bug

My game on Uma Musume Pretty Derby will be stuck on the now loading screen and won't do anything else. Help will be grateful!
<img width="599" alt="Screenshot 2023-12-16 at 11 05 23 AM" src="https://github.com/PlayCover/PlayCover/assets/154011347/0dc59b63-0971-43ee-94b1-6771f4065d37">

### Steps to reproduce

1. Closing and opening the game. 
2. Having to sign it with my Apple ID 
3. Waiting for the tutorial to start 
4. Not starting and stuck on the now loading screen for hours 

### Expected behaviour

For the screen to start the tutorial. 

### What version of PlayCover are you using?

3.0.0 beta 2

### What version of macOS are you using?

Ventura (macOS 13)

### Issue Language

- [X] Yes my issue is written in English

**Comment by sfc0169** [comment]
In my case, when I launch the app, it starts in an enlarged state and I can't revert it back.
![Uploading スクリーンショット 2023-12-26 10.31.16.png…]()

**Comment by tatehiroto** [comment]
To prevent getting stuck, follow these steps:

1. Adjust Graphics Settings:

・Open Uma Musume's settings.

・Tap "Graphics."

・Under "Device," select "iPhone 13 Pro Max A15 6GB."

・Under "Adaptive Resolution," choose "Default."

・Fixed window issues check

2. Transfer Data from a Cleared Tutorial:

・Ensure you have access to save data where the tutorial has already been completed.

・On the title screen, tap the button in the bottom right corner.

・Select "Data Linkage."

・Follow the prompts to link and transfer the data.

・Once the transfer is finished, tap the title screen again.

・You should now be able to proceed smoothly. 　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　(Since translation is used there may be something wrong)

**Comment by Gabby790** [comment]
Hey thanks it works but now it is now stick on a white screen


<img width="599" alt="Screenshot 2024-01-05 at 8 52 00 PM" src="https://github.com/PlayCover/PlayCover/assets/154011347/8d8c4f5c-de33-4775-86dc-377d55000400">

**Comment by tatehiroto** [comment]
おそらくウイニングライブのチュートリアルのスタックだと思います。チュートリアルのウイニングライブを回避するには、チュートリアルをクリアしたことをデータ転送してください。

**Comment by tatehiroto** [comment]
Perhaps it's the stack from the Winning Live tutorial.　To avoid the winning live of the tutorial, please data transfer that you cleared the tutorial.

**Comment by tatehiroto** [comment]
Please prepare the data you have completed the tutorial with an emulator or smartphone.
